.ci/appveyor.yml

```diff
@@ -47,7 +47,21 @@ install:
     -r requirements.txt -r docs-requirements.txt"
   - ps: "Install-Product node ''"  # Use latest node v5.x.x
   - "npm config set loglevel warn"
+  - "sed -i '/alex/d' package.json"
+  - "sed -i '/coffeelint/d' package.json"
+  - "sed -i '/csscomb/d' package.json"
+  - "sed -i '/docker/d' package.json"
+  - "sed -i '/elm/d' package.json"
```

Comment:
I am a bit confused about `alex` and `elm`. Other than these 2, all changed packages have been removed from `package.json`.